.github/workflows/pr.yml

```
@@ -52,3 +52,17 @@ jobs:
         run: sudo pip install LinkChecker
       - name: Check internal links
         run: linkchecker --ignore-url /search http://localhost:${PORT}
+
+  inclusive-naming-check:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: woke
```

I'd suggest more descriptive step name

```suggestion
      - name: Check inclusive naming
```